workspaces/adr/plugins/search-backend-module-adr/src/collators/DefaultAdrCollatorFactory.ts

```diff
@@ -88,10 +87,6 @@ export type AdrCollatorFactoryOptions = {
    * URL Reader
    */
   reader: UrlReaderService;
-  /**
-   * Token Manager
-   */
-  tokenManager?: TokenManagerService;
```

Comment:
Since we're going to make this a breaking change, lets also make the `auth` param required now - that way we can drop the `createLegacyAuthAdapters` below and completely remove the dependency on the deprecated `backend-common` package

Reply:
Sound good, I've made this change, the sooner we do these things the better IMO. 👍